src/ducks/settings/Rules.styl

```diff
@@ -0,0 +1,2 @@
+.AddRuleButton:first-child:last-child
```

Comment:
I think you can use [`:only-child`](https://developer.mozilla.org/fr/docs/Web/CSS/:only-child)

Reply:
TIL